dockers/MathJaxFonts/Dockerfile

```diff
@@ -23,39 +26,41 @@ RUN apt-get -qq update && apt-get -qqy install \
     woff-tools \
     pkg-config \
     libharfbuzz-dev \
-    libfreetype6-dev || true
-RUN gem install ttfunk --version 1.1.1
+    libfreetype6-dev \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/* \
+ && gem install ttfunk --version 1.1.1
 
 # Download yuicompressor
-RUN mkdir /usr/share/yui-compressor/
-RUN wget "https://github.com/yui/yuicompressor/releases/download/v2.4.8/yuicompressor-2.4.8.jar" -O /usr/share/yui-compressor/yui-compressor.jar
+ADD https://github.com/yui/yuicompressor/releases/download/v2.4.8/yuicompressor-2.4.8.jar /usr/share/yui-compressor/yui-compressor.jar
 
 # Download batik-ttf2svg.jar
-RUN wget "https://archive.apache.org/dist/xmlgraphics/batik/batik-1.7.zip"
-RUN unzip -qq batik-1.7.zip
-RUN mv batik-1.7/batik-ttf2svg.jar /usr/share/java/
+RUN wget "https://archive.apache.org/dist/xmlgraphics/batik/batik-1.7.zip" \
+ && unzip batik-*.zip batik-*/batik-ttf2svg.jar \
+ && mv batik-*/batik-ttf2svg.jar /usr/share/java/ \
+ && rm -r batik-*
 
 # Download and compile ttf2eof (note we add a patch to make it compile)
-RUN wget "https://ttf2eot.googlecode.com/files/ttf2eot-0.0.2-2.tar.gz"
-RUN tar -xzf ttf2eot-0.0.2-2.tar.gz
-RUN sed -i "1s/^/#include <cstddef>/" ttf2eot-0.0.2-2/OpenTypeUtilities.h
-RUN make -C ttf2eot-0.0.2-2/
-RUN mv ttf2eot-0.0.2-2/ttf2eot /usr/bin/
+RUN wget "https://github.com/wget/ttf2eot/archive/v0.0.2-2.tar.gz" -O ttf2eot.tar.gz\
+ && tar -xzf ttf2eot.tar.gz \
+ && sed -i "1s/^/#include <cstddef>/" ttf2eot-*/OpenTypeUtilities.h \
+ && make -C ttf2eot-*/ \
+ && mv ttf2eot-*/ttf2eot /usr/bin/ \
+ && rm -r ttf2eot*
```

Comment:
Not necessary for the image to perform as intended, no. The goal is again to clean up all the build-time resources once the desired command has been installed, to keep the size of the layer down.